execution/db-bootstrapper/src/bin/db-bootstrapper.rs

```diff
@@ -64,6 +64,7 @@ fn main() -> Result<()> {
         // When not committing, we open the DB as secondary so the tool is usable along side a
         // running node on the same DB. Using a TempPath since it won't run for long.
         tmpdir = TempPath::new();
+        tmpdir.create_as_dir()?;
```

Comment:
cc @msmouse. Seem there's some semantic change to the `TempPath` that caused this line to fail in the test.